### DISCUSSION TITLE
Custom Icons

### DISCUSSION THREAD

**Discussion by E0697** [Kunzisoft/KeePassDX#1238] [Q&A] [answered]
Login icons provided on the website are in svg format, which isn't supported hy the keepass format. There's plenty of online conversion websites, but the result is not always satisfying. Do you have any suggestions regarding this?

**Comment by J-Jamet** [comment, marked as answer]
Pretty sure that the on-the-fly conversion to PNG works when clicking on an icon on keepassdx.com.
I use it all the time

**Reply by E0697** [reply]
Just tried. It works perfectly!
I used to long press and download and the format would be svg.
Thank you!
